content/en/synthetics/private_locations/configuration.md

```diff
@@ -99,6 +99,11 @@ Discard SSL errors when private location is using a proxy to send requests to Da
 **Default**: `10`<br>
 Maximum number of tests executed in parallel.
 
+`maxNumberMessagesToFetch`
+: **Type**: Number <br>
+**Default**: `10`<br>
+Maximum number of tests fetched from Datadog.
```

Comment:
Just flagging the mismatch between the parameter's name and description. Could you confirm if this is intentional?

Reply:
message is indeed tests

Reply:
good to go :)